tests/python3_f_strings/jsfmt.spec.js

```diff
@@ -1 +1 @@
-run_spec(__dirname, ["python"], { pythonVersion: "3" });
+run_spec(__dirname, ["python"], ">=3.6");
```

Comment:
might as well... python 3.5 still sees some use, since it's what some of the active ubuntu LTS versions still ship

Reply:
Elasticbeanstalk (AWS) is stuck to python 3.4, I'd keep support for 3.4+ if possible